examples/oneapi/cpp/makefile_lnx

```diff
@@ -121,35 +122,28 @@ LOPTS := -L"$(DAAL_PATH)" \
          -L"$(TBB_PATH)" \
          $(LIBS)
 
-RES_DIR=_results/$(compiler)_$(_IA)_$(threading)_$(RES_EXT)
+RES_DIR=_results/$(compiler)_intel64_$(threading)_$(RES_EXT)
```

Comment:
From sln files https://github.com/oneapi-src/oneDAL/blob/master/examples/daal/cpp_sycl/daal_win.sln.tpl need to  remove support ia32?